containers/Makefile.am

```diff
@@ -0,0 +1,39 @@
+
+KUBERNETES_DIR = $(srcdir)/containers/kubernetes
+
+CLEANFILES += $(KUBERNETES_DIR)/rpms
+
+kubernetes-container:
+	rm -rf $(KUBERNETES_DIR)/rpms && mkdir -p $(KUBERNETES_DIR)/rpms
+	cd $(KUBERNETES_DIR)/rpms && ../../../tools/make-rpms
+	sudo docker build -t cockpit/kubernetes $(KUBERNETES_DIR)
```

Comment:
Having this use the build-arg --build-arg OFFLINE=1 would be nice. To avoid the slow dnf update. Of course that means changing the base image to use cockpit/base. We could just have an alternate file ei: testDockerFile that we can use both in tests and for this.